cog/api/gift.py

```
@@ -0,0 +1,64 @@
+            response = requests.post(url, headers=self.headers, json=payload, timeout=5)
+            response.raise_for_status()
+            return response.json()
+        except requests.RequestException as e:
+            return {"result": "internal server error", "status": 500, "error": str(e)}
+        except Exception as e:
+            return {"result": "internal server error", "status": 500, "error": str(e)}
+
+    def _gen_msg(self, gift_type: str, gift_amount: int) -> str:
+        embed = {
+            "title": f"你收到了 {gift_amount} {gift_type}!",
+            "color": 3447003,  # （藍色）
```

Ruff

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)